fdbserver/DataDistribution.actor.cpp

```diff
@@ -3960,14 +3960,67 @@ ACTOR Future<vector<std::pair<StorageServerInterface, ProcessClass>>> getServerL
 // to a sorted PID set maintained by the data distributor. If now no storage server exists, the new Process ID is 0.
 ACTOR Future<Void> updateNextWigglingStoragePID(DDTeamCollection* teamCollection) {
 	state ReadYourWritesTransaction tr(teamCollection->cx);
-	state Value writeValue;
+	state Value writeValue = LiteralStringRef("");
```

Comment:
nit: the codebase is slowly moving towards suffix notation over this macro, i.e. `""_sr`.

Reply:
Changed to ""_sr format.